src/main/java/com/mparticle/kits/RokoMobiKit.java

```diff
@@ -0,0 +1,260 @@
+        return null;
+    }
+
+    @Override
+    public List<ReportingMessage> leaveBreadcrumb(String s) {
+        return null;
+    }
+
+    @Override
+    public List<ReportingMessage> logError(String s, Map<String, String> map) {
+        return null;
+    }
+
+    @Override
+    public List<ReportingMessage> logException(Exception e, Map<String, String> map, String s) {
+        return null;
+    }
+
+    @Override
+    public List<ReportingMessage> logEvent(MPEvent mpEvent) {
+        Log.e(getName(), mpEvent.getEventName());
+        Event rokoEvent = new Event(mpEvent.getEventName());
+        Map<String, String> eventInfo = mpEvent.getInfo();
+        for (Map.Entry<String, String> entry : eventInfo.entrySet()) {
+            rokoEvent.set(entry.getKey(), entry.getValue());
+        }
+        RokoLogger.addEvents(rokoEvent);
+        return null;
+    }
+
+    @Override
+    public List<ReportingMessage> logScreen(String s, Map<String, String> map) {
+        return null;
+    }
+
+    @Override
+    public void checkForDeepLink() {
+        Activity currentActivity = getCurrentActivity().get();
+        if (currentActivity == null) {
+            return;
+        }
+
+        final Intent intent = currentActivity.getIntent();
+
+        RokoLinks.getByVanityLinkCmd(currentActivity, intent, new CallbackVanityLink() {
+            @Override
+            public void success(final ResponseVanityLink res) {
+                try {
+                    DeepLinkResult deepLinkResult = new DeepLinkResult()
+                            .setLink(intent.getDataString())
+                            .setServiceProviderId(getConfiguration().getKitId())
+                            .setParameters(new JSONObject(Json.serialize(res.data)));
+                    getKitManager().onResult(deepLinkResult);
+                } catch (JSONException e) {
+                    e.printStackTrace();
+                }
+            }
+
+            @Override
+            public void failure(final String error) {
+                getKitManager().onError(new DeepLinkError()
+                        .setMessage(error)
+                        .setServiceProviderId(getConfiguration().getKitId()));
+
+            }
+        });
+    }
+
+    @Override
+    public void setInstallReferrer(Intent intent) {
+        RokoLinks.handleDeepLink(intent);
+    }
+
+    @Override
+    public boolean willHandlePushMessage(Intent intent) {
+        return intent != null && intent.getStringExtra(RK_MSG_ID) != null;
+    }
+
+    @Override
+    public void onPushMessageReceived(final Context context, Intent intent) {
+        final Bundle bundle = intent.getExtras();
+        ThreadUtils.runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+                if (bundle.containsKey(RokoPushConstants.EXTRA_OVERLAY_ID)) {
+                    PushNotificationHelper.pulNotificationSent(bundle.getString(RokoPushConstants.EXTRA_OVERLAY_ID));
```

Comment:
this line doesn't compile "pulNotificationSent"

Reply:
we'll followup - cc @alexsobolev